### PR TITLE
Added funders logos

### LINKED ISSUE #114: Add funder logos to About page
@ldecoursy created a list with Jessie Keating (github?) of funders. We will add these logos to the bottom of the about page

Page divider Heading: "Network Supporters" 

 - [x] Alfred P. Sloan Foundation
 - [x] The Helmsley Charitable Trust
 - [x] Ford Foundation
 - [x] National Science Foundation
 - [x] Knight Foundation
 - [x] MacArthur Foundation
 - [x] Open Society Foundations
 - [x] Open Tech Fund
 - [x] Ontario Trillium Foundation
 - [x] C.S. Mott Foundation
 - [x] Institute for Museum and Library Services
 - [x] Susan Crowne Exchange
 - [x] Canadian Internet Registration Authority
 - [x] Chicago Community Trust
 - [x] New York Community Trust
 - [x] The Rockefeller Foundation
 - [x] Siegel Family Endowment
 - [x] Robert R. McCormick Foundation
 - [x] Illinois Science & Energy Foundation
 - [x] Altman Foundation
 - [x] UN Women
 - [x] Irvine Foundation
 - [x] Bill and Melinda Gates Foundation

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #114

### CODE REVIEW

**Comment by mmmavis** [comment]
@alanmoo @gvn I can't tell what's causing Travis failure. Is it related to changes in this PR?

**Comment by gvn** [comment]
@mmmavis Travis was failing for everything because of an issue with Node downloads. I put the details in the production channel. 🎉

**Comment by gvn** [comment]
Oh, and it looks like you have a pug formatting issue.

File: source/pug/views/about.pug [hunk 1]
```diff
@@ -31,4 +31,71 @@ block content
...
+    .row.big-section.justify-content-around
+      .col-12.col-md-11.col-md-offset-1
+        .row.mb-5
+          .col-12
+            .slanty-background
+              h3.h2-headings-white Network Supporters
+        .row
+          .col-12
+            .row
+              .col-6.col-md-3.mb-5.px-3.d-flex.align-items-center.justify-content-center
```
Comment (gvn) [inline source/pug/views/about.pug]: Can you do these as a mixin to DRY this out?

**Comment by gvn** [comment]
Sorry, I should have specified a Pug mixin. For example:

```diff
diff --git a/source/pug/views/about.pug b/source/pug/views/about.pug
index 6770895..cda56f4 100644
--- a/source/pug/views/about.pug
+++ b/source/pug/views/about.pug
@@ -1,7 +1,11 @@
extends ../templates/master.pug

mixin funderLogo(uri)
  .col-6.col-md-3.mb-5.px-3.d-flex.align-items-center.justify-content-center
    img.w-75(src=`${env.virtualRoot}${uri}`)

block centerNav
  p About

block content
  .container.py-5
@@ -38,12 +42,11 @@ block content
            .slanty-background
              h3.h2-headings-white Network Supporters
        .row
          .col-12
            .row
              .about-funder-logo
                img.w-75(src=`${env.virtualRoot}_images/about/alfred-p-sloan-foundation.jpg`)+funderLogo(`_images/about/alfred-p-sloan-foundation.jpg`)
```

**Pull request merged**